mkdocs.yml

```
@@ -13,7 +13,7 @@ validation:
   not_found: warn
   absolute_links: ignore
 # fails to build on warnings
-strict: true
+# strict: true
```

```suggestion
strict: true
```
Is there a reason you pushed this? I know locally it does not usually work properly, but on building on GitHub it should be fine.

---

as discussed in the channel this was a error on my side so i could test run it locally, and by mistake i merged it.
thnx for the revert